### PR TITLE
chore: adds a script to fetch all gcp projects

### PR DESCRIPTION
**Pull request by sameer-in**
- adding script to fetch all gcp projects

### CODE REVIEW

File: resources/fetch-gcp-projects.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  # List subfolders and call this function recursively
+  local subfolders=$(gcloud resource-manager folders list --folder=$folder_id --format="value(name)")
+  for subfolder in $subfolders; do
+    list_projects_recursive "$subfolder"
+  done
+}
+
+# List projects under the root organization
+org_id="933620940614"
```
Comment (haresh-suresh) [inline resources/fetch-gcp-projects.sh:18]: Let's remove our test orgID from the script

Comment (sameer-in) [reply resources/fetch-gcp-projects.sh]: done. removed it.

**Review by haresh-suresh — APPROVED** [review]

**Review by wideawakening — DISMISSED** [review]
added organization ID as parameter and usage.

thanks for the addition team!

File: examples/organization-org_compliance/README.md [hunk 2]
```diff
@@ -16,10 +16,15 @@ This example deploys Secure for Cloud into a GCP organizational account.
 ## Prerequisites
 
 1. Configure [Terraform **GCP** Provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs)
-2. Following **roles** are required in your GCP organization/project credentials
+2. Run the script at `resources/fetch-gcp-projects.sh`. copy the output and provide it as input in the module
```
Comment (haresh-suresh) [inline examples/organization-org_compliance/README.md:19]: Maybe we can say :
Run the script at `resources/fetch-gcp-projects.sh <orgID>` ?

Comment (sameer-in) [reply examples/organization-org_compliance/README.md]: Done. Also added link to documentation

**Review by haresh-suresh — APPROVED** [review]

**Review by tembleking — APPROVED** [review]

**Pull request merged**